cmd/start/start.go

```diff
@@ -92,6 +93,21 @@ func Run(bootstrapConfigPath string) {
 		propeller.AddListener(listener)
 	}
 
+	// init service registry
+	registryConf := discovery.GetServiceRegistry(context.Background())
+	if registryConf != nil && registryConf.Enable {
+		serviceRegistry, err := registry.InitRegistry(registryConf)
+		if err != nil {
+			log.Fatalf("create service registry failed: %v", err)
```

Comment:
How about use ERR level? The `Fatalf` will cause process exiting, but here I think it's not necessary to let process killed.